learners/training_calendar.md

```diff
@@ -83,6 +83,12 @@ New trainings are added to this calendar on a quarterly basis.
 
 <hr>
 
+#### [September 19-22, 2023](https://www.eventbrite.com/e/online-instructor-training-19-22-september-2023-tickets-669387976457?aff=oddtdtcreator)
```

Comment:
```suggestion
#### [September 19-22, 2023](https://www.eventbrite.com/e/online-instructor-training-19-22-september-2023-tickets-669387976457)
```

Reply:
I'm not sure what is going on with this link -- it works either way -- but I think it should probably be formatted like the others.